testing/trino-tests/src/test/java/io/trino/tests/tpch/TestTpchConnectorTest.java

```diff
@@ -61,6 +61,7 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
                 return false;
 
             case SUPPORTS_TOPN_PUSHDOWN:
+            case SUPPORTS_DEREFERENCE_PUSHDOWN:
```

Comment:
should be redundant, since tpch doesn't support row type

add `SUPPORTS_ROW_TYPE -> false` instead